examples/conditional-artifacts.yaml

```diff
@@ -0,0 +1,81 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Workflow
+metadata:
+  generateName: conditional-artifacts-
```

Comment:
can you add the testing labels and annotations please? make sure to write a good description